sites/header/includes/header.html

```diff
@@ -0,0 +1,39 @@
+<header>
+	<div id="wb-bnr" class="container">
+		<div class="row">
+			{% if page.altLangPage %}
+				{% include language/inc-languagetoggle.html %}
+			{% endif %}
+			{% include headers-includes/brand.html %}
+			{% unless page.nositesearch %}
+				{% include headers-includes/sitesearch.html %}
+			{% endunless %}
+		</div>
+	</div>
+	<hr>
+	<div class="container">
+		<div class="row">
+			{%- unless page.nomenu -%}
+				{%- if page.auth -%}
+					<div class="col-sm-8">
+						{% include gcweb-menu/sitemenu.html %}
+					</div>
+				{%- else -%}
+					<div class="col-sm-12">
+						{% include gcweb-menu/sitemenu.html %}
+					</div>
+				{%- endif -%}
+			{%- endunless -%}
+			{%- if page.auth and page.nomenu -%}
+				<div class="col-sm-12">
+					{% include authentication/authentication.html %}
+				</div>
+			{%- elsif page.auth -%}
+				<div class="col-sm-4">
```

Comment:
@Ricokola @duboisp At what point did we decide that the Sign in button would end up on a different line than the menu on XS view? I agree that the Signed on / Signed off should, but not the contextual Sign in button.

See the screenshot:
<img width="767" alt="Screen Shot 2022-05-17 at 5 11 19 PM" src="https://user-images.githubusercontent.com/4353199/168910888-eb06769d-274c-4c1e-8987-8d396d94eab7.png">


If the problem was space, the remedy to this is to maybe try to use `.row-no-gutters` and play with the columns, like so:
<img width="762" alt="Screen Shot 2022-05-17 at 4 51 10 PM" src="https://user-images.githubusercontent.com/4353199/168910698-7fe7b8d7-846b-434a-b28a-b277f74a6dbc.png">

@Ricokola @duboisp But I think the biggest problem remains, for XS and SM, that the menu once open actually won't go wider than it's defined column, while in fact it should take up 100% until 991 px wide included. For example, right now the menu looks like this on small view port while in previous version it was taking the full-width:
<img width="768" alt="Screen Shot 2022-05-17 at 5 15 55 PM" src="https://user-images.githubusercontent.com/4353199/168911498-671a68bf-19a0-45d9-a5d9-ef1d987628c9.png">

This to me doesn't too bad for SM, but for XS if we apply my change requested to have Menu and Sign in side-by-side, we end up with this which isn't acceptable:
<img width="319" alt="Screen Shot 2022-05-17 at 5 19 36 PM" src="https://user-images.githubusercontent.com/4353199/168911931-ec3e12ae-2d3b-42b7-b585-e8ee97b1f76a.png">

There me be a CSS answer to this, but we'll need to discuss.